Task10_Auth/src/test/java/GetUsersTest.java

```diff
@@ -0,0 +1,69 @@
+import org.junit.jupiter.api.Test;
+import reseponsesDTO.UsersDto;
+
+import java.util.ArrayList;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class GetUsersTest {
```

Comment:
Common comment for this class:
1. You're using not interface as type (List vs ArrayList)
2. Tests will test nothing if there is no users in app

Reply:
2. Added BeforeTest which create the user if it's not exist

Reply:
1. Chhanged from ArrayList<UserDto> to List<UserDto>